iot-e2e-tests/common/src/main/java/com/microsoft/azure/sdk/iot/common/serviceclient/ExportImportCommon.java

```diff
@@ -30,9 +31,9 @@
 
 public class ExportImportCommon
 {
-    private static final long IMPORT_EXPORT_TEST_TIMEOUT = 3 * 60 * 1000; //3 minutes
-    private static final long IMPORT_JOB_TIMEOUT = 1 * 60 * 1000; //1 minute
-    private static final long EXPORT_JOB_TIMEOUT = 1 * 60 * 1000; //1 minute
+    private static final long IMPORT_EXPORT_TEST_TIMEOUT = 8 * 60 * 1000; //3 minutes
```

Comment:
>3 [](start = 76, length = 1)

I think it's safe to delete the command....it's pretty easy to understand these math here

Reply:
Sure, I'll get rid of the comments